### ISSUE TITLE
If a yara rule is in the directory, the unpacker will end with an error

### ISSUE THREAD

**Issue by mienkofax** [avast/retdec#969]
If I run `retdec-unpacker` from the directory where the `*.yar` or `*.yara` files is located, the program will end with an error. Small example follows. 

```bash
$ ls
36b28625c4b9ebcdffd56b1ddabd0ca8f8325db520990c3e5890dd562dd2d823

$ retdec-unpacker 36b28625c4b9ebcdffd56b1ddabd0ca8f8325db520990c3e5890dd562dd2d823
[UPX] Detected DIRECT JUMP scrambler unpacking stub based on signature & metadata.
[UPX] Started unpacking of file '36b28625c4b9ebcdffd56b1ddabd0ca8f8325db520990c3e5890dd562dd2d823'.
[UPX] Detected NRV2B unpacking stub based on metadata.
[UPX] Detected filter 0x26 with parameter 0x12 based on UPX metadata.
[UPX] Unpacked data based on UPX metadata.
[UPX] Original header found at address 0x44be50 in extra data.
[UPX] Import hints address 0x44a000 found in extra data.
[UPX] Relocations hints address 0x44a758 found in extra data.
[UPX] Original TLS directory found at RVA 0x5fab0 with size 0x18.
[UPX] Original entry point address set to 0x310d6.
[UPX] Original load configuration directory found at RVA 0x57f00 with size 0xa0.
[UPX] Original resources directory found at RVA 0x6b000 with size 0x3db2f8.
[UPX] Original certificates directory found at offset 0x255a00 with size 0x20f0.
[UPX] Packed file has overlay with size of 0x152b7 bytes. Copying into unpacked file.
[UPX] Successfully unpacked '36b28625c4b9ebcdffd56b1ddabd0ca8f8325db520990c3e5890dd562dd2d823'!

$ cp ../yara-rule.yar .
$ retdec-unpacker 36b28625c4b9ebcdffd56b1ddabd0ca8f8325db520990c3e5890dd562dd2d823
retdec-unpacker: compiler.c:701: yr_compiler_get_rules: Assertion `compiler->errors == 0' failed.
Aborted (core dumped)
```

**Comment by metthal** [comment]
This should be fixed since 4878585d1a51e7b4c28c8eae7ed7bb2cbc531f93, could you please check that you have this commit in your local build and try again?

**Comment by mienkofax** [comment]
I used an older version, in the new version it works without problems, thank you.
